src/main/resources/application.yml

```diff
@@ -20,3 +21,92 @@ rs256:
 spring:
   application:
     name: Service Auth Provider
+  cloud:
+    propertiesvolume:
+      paths: /mnt/secrets/microservicekey-cmcLegalFrontend,
+        /mnt/secrets/microservicekey-cmcClaimStore,
+        /mnt/secrets/microservicekey-divorce,
+        /mnt/secrets/microservicekey-divorce-frontend,
+        /mnt/secrets/microservicekey-divorceCcdSubmission,
+        /mnt/secrets/microservicekey-divorceCcdValidation,
+        /mnt/secrets/microservicekey-divorceDocumentUpload,
+        /mnt/secrets/microservicekey-divorceDocumentGenerator,
+        /mnt/secrets/microservicekey-draftStoreTests,
+        /mnt/secrets/microservicekey-platformJobScheduler,
+        /mnt/secrets/microservicekey-reference,
+        /mnt/secrets/microservicekey-sscs,
+        /mnt/secrets/microservicekey-sscs-bulkscan,
+        /mnt/secrets/microservicekey-sscs-tribunals-case,
+        /mnt/secrets/microservicekey-probate-frontend,
+        /mnt/secrets/microservicekey-probate-backend,
+        /mnt/secrets/microservicekey-send-letter-consumer,
+        /mnt/secrets/microservicekey-send-letter-tests,
+        /mnt/secrets/microservicekey-em-gw,
+        /mnt/secrets/microservicekey-finrem-document-generator,
+        /mnt/secrets/microservicekey-finrem-notification-service,
+        /mnt/secrets/microservicekey-finrem-case-orchestration,
+        /mnt/secrets/microservicekey-finrem-payment-service,
+        /mnt/secrets/microservicekey-finrem-ccd-data-migrator,
+        /mnt/secrets/microservicekey-jui-webapp,
+        /mnt/secrets/microservicekey-pui-webapp,
+        /mnt/secrets/microservicekey-coh-cor,
+        /mnt/secrets/microservicekey-bulk-scan-processor,
+        /mnt/secrets/microservicekey-bulk-scan-processor-tests,
+        /mnt/secrets/microservicekey-bulk-scan-orchestrator,
+        /mnt/secrets/microservicekey-bar-api,
+        /mnt/secrets/microservicekey-api-gw,
+        /mnt/secrets/microservicekey-em-annotation-app,
+        /mnt/secrets/microservicekey-em-npa-app,
+        /mnt/secrets/microservicekey-cet,
+        /mnt/secrets/microservicekey-cet-frontend,
+        /mnt/secrets/microservicekey-fpl-case-service,
+        /mnt/secrets/microservicekey-iac,
+        /mnt/secrets/microservicekey-em-stitching-api,
+        /mnt/secrets/microservicekey-em-ccd-orchestrator,
+        /mnt/secrets/microservicekey-ccpay-bubble,
+        /mnt/secrets/microservicekey-dg-template-management,
+        /mnt/secrets/microservicekey-dg-docassembly-api,
+      aliases:
+        microservicekey-cmcLegalFrontend: MICROSERVICEKEYS_CMC_LEGAL_FRONTEND
+        microservicekey-cmcClaimStore: MICROSERVICEKEYS_CMC_CLAIM_STORE
```

Comment:
You need to prefix your aliases with the keyvault, e.g. 's2s.microservicekey-cmcClaimStore'